create_derivatives/tests.py

```diff
@@ -27,9 +26,9 @@ def test_check_dir_exists(self):
                 p in str(context.exception), "Directory was not found in exception")
 
     def test_matching_files(self):
-        MATCHING_FIXTURE_FILEPATH = join("create_derivatives", "fixtures", "matching")
-        MATCHING_SOURCE_DIR = join("/", "matching")
-        if isdir(MATCHING_SOURCE_DIR):
+        MATCHING_FIXTURE_FILEPATH = Path("create_derivatives").joinpath("fixtures").joinpath("matching")
+        MATCHING_SOURCE_DIR = Path("matching")
```

Comment:
These two things are not equivalent - you've replaced an absolute path with a relative one.